File: server/groute/src/main/java/com/ssafy/groute/dto/Account.java
package com.ssafy.groute.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Setter
@Getter
@NoArgsConstructor
@AllArgsConstructor
public class Account {
    private int id;
    private int spentMoney;
    private String description;
    private String category;
    private String cateImg;
    private int userPlanId;
    private String type;
    private int day;
}

File: client/Groute/app/src/main/java/com/ssafy/groute/src/dto/Account.kt
package com.ssafy.groute.src.dto

data class Account(
    val categoryId:Int,
    val categoryName: String,
    val day:Int,
    val description: String,
    val id: Int,
    val img:String,
    val userPlanId: Int,
    val spentMoney: Int,
    val type:String
){
    constructor(categoryId: Int,day: Int,description: String,userPlanId: Int,spentMoney: Int,type: String):this(categoryId,"",day, description, 0, "", userPlanId, spentMoney, type)
}